src/main/java/Yuki.java

```diff
@@ -0,0 +1,87 @@
+import java.util.Scanner;
+public class Yuki {
+
+    // create array of Tasks (use of polymorphism)
+    private static Task[] tasks = new Task[100];
+    private static int taskCount = 0;
+
+    public static void addTask(Task t) {
+        tasks[taskCount] = t;
+        taskCount++;
+        System.out.println(t);
+    }
+
+    public static void reportNumberOfTasks() {
+        System.out.println("Now you have " + taskCount + " tasks in the list.");
+    }
+
+    public static void main(String[] args) {
```

Comment:
The main method here is quite long and handles multiple responsibilities. Maybe it could be refactored into smaller methods to improve readability.